doc/02-installation.md

```diff
@@ -358,7 +358,7 @@ to determine where to find the plugin binaries.
 
 <!-- {% if debian or ubuntu or raspbian %} -->
 <!-- {% if not icingaDocs %} -->
-#### Debian / Ubuntu / Raspbian
+#### Debian / Ubuntu / Raspbian / Raspberry Pi OS
```

Comment:
> refs https://github.com/Icinga/package-installation-docs/issues/3

Wouldn't it make sense to clean this up in one go if you're touching these places already?

Reply:
I.e. remove " / Raspbian" and its .md file?

Reply:
Yes. We stopped building packages for 32 bit Raspbian/Raspberry Pi OS in favor for arm64 Debian packages which can also be used on Raspberry Pi OS 64 bit, so the changes to the documentation should reflect that as well.